src/main/java/gregtech/api/metatileentity/MetaTileEntity.java

```diff
@@ -970,9 +970,6 @@ private <T> void transferToNearby(Capability<T> capability, BiConsumer<T, T> tra
             T otherCap = tileEntity.getCapability(capability, nearbyFacing.getOpposite());
             //use getCoverCapability so item/ore dictionary filter covers will work properly
             T thisCap = getCoverCapability(capability, nearbyFacing);
-            if (otherCap == null || thisCap == null) {
```

Comment:
rebase on top of master

Reply:
My git is ***really rusty***, thanks I'll hopefully get that sorted tomorrow